<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ticketDialog</class>
 <widget class="QDialog" name="ticketDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_nom_film">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nom du film :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_salle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Salle</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_horaire">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Horaire</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_abo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Abo</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_nb_places">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de places</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_duree">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Dur√©e</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Fermer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_print">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Imprimer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
